get_started/simple_pipeline.md

```diff
@@ -0,0 +1,247 @@
+```elixir
+defmodule Hello do
+
+  use Membrane.Pipeline
+
+  @impl true
+  def handle_init(path_to_mp3) do
+  end
+  
+end
+```
+
+The main purpose of `handle_init` callback is to prepare our pipeline. Preparing among others means that we need to specify all its elements as children and set-up linkings between them. All other options that can be specified if needed you can find in `Membrane.ParentSpec` [docs](https://hexdocs.pm/membrane_core/Membrane.ParentSpec.html).
+
+>If concept of callbacks and behaviours is new to you, you should probably take some time to read about OTP in Elixir (especially a part starring GenServer and Supervisor). You can find proper guide [here](https://elixir-lang.org/getting-started/mix-otp/agent.html)
+
+#### Elements
+
+The elements we'd like to use to play our mp3 will be:
+
+ - `Membrane.File.Source` - the Source module form Membrane file plugin that will read a file.
+ - `Membrane.MP3.MAD.Decoder` - an mp3 decoder based on [MAD](https://www.underbit.com/products/mad/)
+ - `Membrane.FFmpeg.SWResample.Converter` - a converter based on [FFmdeg SWResample](https://ffmpeg.org/doxygen/trunk/group__lswr.html#details). We'll be needing it to resample our row audio stream from 24 to 16 bits.
+ - `Membrane.Portaudio.Sink` - A Sink module that will be playing our music with [Portaudio](http://www.portaudio.com)
+
+> As you can see all elements we're using are not a part of `membrane_core`, but can be found in separate libraries. You can find a list of packages provided by Membrane team [here](https://membraneframework.org/guide/v0.7/packages.html). You can also learn how to write your own Element.
+
+The full children declaration for our player will look like that:
+
+```elixir
+children = %{
+  file: %Membrane.File.Source{location: path_to_mp3},
+  decoder: Membrane.MP3.MAD.Decoder,
+  converter: %Membrane.FFmpeg.SWResample.Converter{
+    output_caps: %Membrane.Caps.Audio.Raw{
+	  format: :s16le,
+	  sample_rate: 48000,
+	  channels: 2
+	}
+  },
+  portaudio: Membrane.PortAudio.Sink
+}
+```
+
+The keys in that keyword list are just a names we gave to elements. We're going to need them when linking.
+
+#### Linking elements
+
+Now we should link them in proper order. Each membrane element is one of three types: Source, Sink or Filter. The main difference is that Source provides only output pads, Sink only input and Filter both input and output. That means Source element can only start pipelines (it's not prepared to receive any data from other elements), Sink can only end pipeline (it will not send any data to succeeding elements), and Filters can be in the middle (it will receive, process and send data further). In our case a links declaration will look like that: 
```

Comment:
```suggestion
Now we should link them in the proper order. Each Membrane Element can be one of three types: Source, Sink or Filter. The main difference is that Source provides only output pads, Sink only input and Filter both input and output pads. That means only a Source element start pipelines (it's not prepared to receive any data from other elements), Sink can only end pipelines (it will not send any data to subsequent elements), and Filters can be in the middle (they receive, process and send data further). In our case the links declaration will look like this: 
```